### PR TITLE
Initial step UI implementations and sample app

### PR DESCRIPTION
**Pull request by nategbrown9**
Initial implementation of UI for Instruction, Choice, and Text based questions.

### CODE REVIEW

File: androidApp/src/main/res/raw/sample_assessment.json [hunk 1]
```diff
@@ -18,7 +18,40 @@
...
+    },
+    {
+      "identifier": "step2",
+      "type": "instruction",
+      "title": "Animating image instruction",
+      "detail": "Here are the details for this instruction.",
+      "image"  : {
+        "type":"fetchable",
+        "imageName" : "crf_stair_step_start_animation"
+      }
```
Comment (syoung-smallwisdom) [inline androidApp/src/main/res/raw/sample_assessment.json:36]: So... this won't work for iOS. An animating image must be defined with a series of image names using the `AnimatedImage` serializable class. This is not compatible JSON.

File: androidApp/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -3,16 +3,15 @@
     <application
                  android:allowBackup="true"
                  android:supportsRtl="true"
-                 android:theme="@style/AppTheme">
+                 android:theme="@style/BlueberryTheme">
```
Comment (syoung-smallwisdom) [inline androidApp/src/main/AndroidManifest.xml:6]: Note: color theme serialization has not been implemented yet.

Comment (nategbrown9) [reply androidApp/src/main/AndroidManifest.xml:6]: This is using Android's built-in theming mechanism.

File: androidApp/src/main/res/raw/sample_assessment.json (hunk 1 above)
Comment (nategbrown9) [reply androidApp/src/main/res/raw/sample_assessment.json:36]: On Android an AnimationDrawable is usually defined in a Drawable xml file which contains the list of images and any animation configuration. I suppose we could manually do that in code.

**Pull request merged**